src/com/pentalog/pentastagiu/homework5/circle/Circle.java

```diff
@@ -0,0 +1,28 @@
+package com.pentalog.pentastagiu.homework5.circle;
+
+public class Circle {
+
+    private Double radius;
+    private final Double PI = 3.14;
```

Comment:
E ok si asa, dar in clasa Math ai helpere pentru calcule matematice, printre care si valoarea lui PI